cartography/intel/gcp/label.py

```diff
@@ -54,20 +53,28 @@ def _load_labels_tx(tx: neo4j.Transaction, data: List[Dict], update_tag: int) ->
         ingest_label,
         data=data,
         update_tag=update_tag,
+        GCP_PROJECT_ID=common_job_parameters['GCP_PROJECT_ID'],
+        WORKSPACE_ID=common_job_parameters['WORKSPACE_ID'],
     )
 
 
-def cleanup_labels(neo4j_session: neo4j.Session, common_job_parameters: Dict) -> None:
-    logger.info("Cleaning Labels....")
+def cleanup_labels(neo4j_session: neo4j.Session, common_job_parameters: Dict, service_name: str) -> None:
+    logger.info(f"Cleaning Labels for {service_name}")
     run_cleanup_job('gcp_labels_cleanup.json', neo4j_session, common_job_parameters)
 
 
 def sync_labels(
     neo4j_session: neo4j.Session, data: List[Dict], update_tag: int, common_job_parameters: Dict,
-    service: str,
+    service_name: str, service_label: str,
 ) -> None:
     if len(data) > 0:
         labels_list = get_labels_list(data)
         if len(labels_list) > 0:
-            logger.info(f"Loading Labels for {service}")
-            load_labels(neo4j_session, labels_list, update_tag)
+            logger.info(f"Loading Labels for {service_name}")
+            common_job_parameters['service_label'] = service_label
+            load_labels(neo4j_session, labels_list, update_tag, common_job_parameters)
+            cleanup_labels(neo4j_session, common_job_parameters, service_name)
+            try:
+                del common_job_parameters["service_label"]
```

Comment:
Should we delete `service_name` as well?

Reply:
yes because every cleanup job has separate `service_name`